specification/paths/CarrierContracts.json

```diff
@@ -181,6 +192,17 @@
                             "currency",
                             "name"
                           ]
+                        },
+                        "relationships": {
+                          "properties": {
+                            "owner": {
+                              "example": {
+                                "links": {
```

Comment:
`links` shouldn't just be an example, it should be properly defined in a schema.

Reply:
The owner relationship is in this case a `"oneOf"` relationship. Which means that our UI doesn't automatically produce an example. If we put it in the example of the schema, that means that post also has links in the example. That's why it was put here as an extra for responses.